pull_request_template.md

```diff
@@ -1,29 +1,34 @@
 # Description
 
-Please include a summary of the changes, including screenshots, if applicable. Please also provide any motivation and context necessary to review these changes. List any dependencies that are required for this change.
+<!-- Please include a summary of the changes, including screenshots, if applicable. Please also provide any motivation and context necessary to review these changes. List any dependencies that are required for this change. -->
 
 ## Type of change
 
-Please delete options that are not relevant.
+<!-- Please delete options that are not relevant.
 
+- [ ] Chore (non-breaking change which has no end user impact)
```

Comment:
can we have a `Docs` one too

Reply:
oops auto merge got me, but i was assuming that docs updates would fall under chore as well.